File: 백준/Bronze/1978. 소수 찾기/소수 찾기.cc
#include <stdio.h>
#include <iostream>

using namespace std;

int main(){
    int N,num;
    int result = 0;
    cin >> N;
    for(int i = 0; i < N; i++){
        cin >> num;
        int count = 0;
        for(int i = 1; i <= num; i++){
            if(num % i == 0){
                count++;
            }
        }
        if(count == 2){
            result++;
        }
    }
    cout << result << endl;
}

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;
import java.util.StringTokenizer;

public class Main {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        StringBuilder sb = new StringBuilder();

        int N = Integer.parseInt(br.readLine());
        int answer = 0;
        StringTokenizer st =new StringTokenizer(br.readLine());
        for (int i = 0; i < N; i++) {
            int num = Integer.parseInt(st.nextToken());
            int count = 0;
            for (int j = 1; j <= num; j++) {
                if (num % j == 0) {
                    count++;
                }
                if (count > 3) {
                    break;
                }
            }
            if (count == 2) {
                answer++;
            }

        }
        sb.append(answer);

        bw.write(sb.toString());
        bw.flush();
    }
}